openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/activity-feed-tab-new.less

```diff
@@ -0,0 +1,402 @@
+        margin-bottom: -6px;
+        border: none;
+        padding: 0px;
+        background: rgba(239, 244, 250, 0.25);
+      }
+      .activity-feed-comment-text {
+        color: #52525b;
+      }
+    }
+  }
+  .activity-feed-card-new-right-panel {
+    background-color: @white;
+    border: none;
+  }
+}
+
+.activity-feed-task {
+  background-color: @grey-1;
+}
+
+.count-loader.ant-skeleton-element .ant-skeleton-button-sm {
+  width: 72px;
+}
+.activity-feed-editor-container-new {
+  background-color: #f5f5f5;
+  border: none;
+  border-radius: 10px;
+  .ql-container {
+    background-color: #f5f5f5;
+    border: none;
+  }
+  .ql-toolbar {
+    background-color: #f5f5f5;
+    border: none;
+    border-bottom: 1px solid #d9d9d9;
+  }
+  .ql-toolbar button {
+    color: #292929;
+  }
+}
+
+.activity-feed-editor-send-btn {
+  color: @white;
+  .ant-btn.send-button {
+    position: absolute;
+    bottom: 8px;
+    right: 8px;
+    width: 34px;
+    height: 34px;
+    padding: 0;
+    border-radius: 50%;
+  }
+}
+.activity-feed-user-name {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  letter-spacing: 0.03em;
+  color: #181d27;
+}
+.reply-card-user-name {
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #373e44;
+}
+.activity-feed-reply-card-footer {
+  margin-left: 45px;
+}
+.active-card {
+  background-color: @blue-9;
+}
+
+.task-tab-custom-dropdown {
+  border-radius: 6px;
+  .task-tab-filter-item {
+    color: #535862;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    &.selected {
+      color: #2e90fa;
+    }
+  }
+
+  .task-filter-icon:hover path {
+    stroke: #2e90fa;
+    stroke-width: 2;
+  }
+  .task-count-container {
+    background: #e9eaeb;
+    border-radius: 16px;
+    width: 30px;
+    height: 22px;
+    display: flex;
+    padding: 2px 8px;
+    justify-content: center;
+    align-items: center;
+    &.active {
+      background-color: #2e90fa !important;
+    }
+
+    .task-count-text {
+      color: #414651;
+      text-align: center;
+      font-family: Inter;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &.active .task-count-text {
+      color: @white;
+    }
+  }
+  .ant-dropdown-menu .ant-dropdown-menu-item {
+    box-shadow: 0px 1px 2px 0px rgba(10, 13, 18, 0.05);
+    padding: 0px;
+    margin: 8px;
+    .active {
+      box-shadow: 0px 1px 2px 0px rgba(10, 13, 18, 0.05);
+      background-color: #f4f4f5;
+      border-radius: 8px;
+      svg path {
+        fill: #2e90fa;
+      }
+    }
+  }
+}
+
+.task-toggle {
+  background: white;
+  border-radius: 8px;
+  padding: 4px;
+  border: 1px solid #dce3ec;
+
+  .ant-segmented-item {
+    color: #71717a;
+    transition: all 0.3s ease;
+
+    &-selected {
+      background: #e6f1fe;
+      color: #005bc4;
+      text-align: center;
+
+      .ant-segmented-item-label {
+        color: #005bc4;
+      }
+      svg path {
+        fill: #005bc4;
+      }
```

Comment:
let's use variables for commonly used colors. It'll be easy to change if required later

Reply:
done